site_scons/site_tools/go_builder.py

```diff
@@ -49,6 +49,26 @@ def _scan_go_file(node, env, _path):
     return includes
 
 
+def get_go_version(output):
+    """Capture only the version after 'go'"""
+    ver_re = re.compile(r'go([0-9\.]+)')
+    groups = ver_re.findall(output)
```

Comment:
FYI there usually isn't a need to compile first
```suggestion
    groups = ver_re.findall(r'go([0-9\.]+)', output)
```